bin/iuf-rpm-upload

```diff
@@ -85,23 +81,23 @@ process(){
     if [[ "${repo_type,,}" == "raw" ]]; then
         # This will load type 'raw' RPM content from the directory at $repo_path into
         # the repository $repo_name.
+        echo "INFO Executing upload script for 'raw' RPM content from $repo_path into $repo_name"
         $SCRIPT_DIR/nexus-upload-repo-raw "$repo_path" "$repo_name"
     elif [[ "${repo_type,,}" == "yum" ]]; then
         # This will load type 'yum' RPM content from the directory at $repo_path into
         # the repository $repo_name.
+        echo "INFO Executing upload script for 'yum' RPM content from $repo_path into $repo_name"
```

Comment:
optimally, "i'm going to do this" messages should be debug, and we should have another INFO message which reports success